week-4/Homework/mandatory/4-groceries.js

```diff
@@ -6,7 +6,12 @@
 let groceriesToBuy = [];
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange Juice",
+  item3: "Rice"
 };
+
+for(let glist in groceryList){
+  groceriesToBuy.push(groceryList[glist]);
```

Comment:
Good usage of for in loop :) . Do you think it would have worked with a method? Which one?